modules/mirror-registry-release-notes.adoc

```diff
@@ -3,179 +3,48 @@
-=== Mirror registry for Red Hat OpenShift 1.3.11
-
-Issued: 2024-04-23
-
-_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.8.15.
-
-The following advisory is available for the _mirror registry for Red Hat OpenShift_:
-
-* link:https://access.redhat.com/errata/RHBA-2024:1758[RHBA-2024:1758 - mirror registry for Red Hat OpenShift 1.3.11]
-
-[id="mirror-registry-for-openshift-1-3-10_{context}"]
-=== Mirror registry for Red Hat OpenShift 1.3.10
-
-Issued: 2023-12-07
-
-_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.8.14.
-
-The following advisory is available for the _mirror registry for Red Hat OpenShift_:
-
-* link:https://access.redhat.com/errata/RHBA-2023:7628[RHBA-2023:7628 - mirror registry for Red Hat OpenShift 1.3.10]
-
-[id="mirror-registry-for-openshift-1-3-9_{context}"]
-=== Mirror registry for Red Hat OpenShift 1.3.9
-
-Issued: 2023-09-19
-
-_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.8.12.
-
-The following advisory is available for the _mirror registry for Red Hat OpenShift_:
-
-* link:https://access.redhat.com/errata/RHBA-2023:5241[RHBA-2023:5241 - mirror registry for Red Hat OpenShift 1.3.9]
-
-[id="mirror-registry-for-openshift-1-3-8_{context}"]
-=== Mirror registry for Red Hat OpenShift 1.3.8
-
-Issued: 2023-08-16
-
-_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.8.11.
-
-The following advisory is available for the _mirror registry for Red Hat OpenShift_:
-
-* link:https://access.redhat.com/errata/RHBA-2023:4622[RHBA-2023:4622 - mirror registry for Red Hat OpenShift 1.3.8]
-
-[id="mirror-registry-for-openshift-1-3-7_{context}"]
-=== Mirror registry for Red Hat OpenShift 1.3.7
-
-Issued: 2023-07-19
-
-_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.8.10.
-
-The following advisory is available for the _mirror registry for Red Hat OpenShift_:
-
-* link:https://access.redhat.com/errata/RHBA-2023:4087[RHBA-2023:4087 - mirror registry for Red Hat OpenShift 1.3.7]
-
-[id="mirror-registry-for-openshift-1-3-6_{context}"]
-=== Mirror registry for Red Hat OpenShift 1.3.6
-
-Issued: 2023-05-30
-
-_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.8.8.
-
-The following advisory is available for the _mirror registry for Red Hat OpenShift_:
+[id="mirror-registry-release-notes-2-0_{context}"]
+== Mirror registry for Red{nbsp}Hat OpenShift 2.0 release notes
 
-* link:https://access.redhat.com/errata/RHBA-2023:3302[RHBA-2023:3302 - mirror registry for Red Hat OpenShift 1.3.6]
+The following sections provide details for each 2.0 release of the mirror registry for Red{nbsp}Hat OpenShift
 
-[id="mirror-registry-for-openshift-1-3-5_{context}"]
-=== Mirror registry for Red Hat OpenShift 1.3.5
+[id="mirror-registry-for-openshift-2-0-0_{context}"]
+=== Mirror registry for Red Hat OpenShift 2.0.0
 
-Issued: 2023-05-18
+Issued: 2024-08-22
```

Comment:
Fwiw, I _think_ the RN team has gone to writing dates out 22 August 2022. Like [this](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-0-ga:~:text=Issued%3A%2017%20January%202023)